plaso/parsers/aws_elb_access.py

```diff
@@ -0,0 +1,312 @@
+# -*- coding: utf-8 -*-
+"""Parser for AWS ELB access logs.
+
+This parser is based on the log format documented at
+https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html
+"""
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.lib import errors
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class AWSELBEventData(events.EventData):
+  """AWS Elastic Load Balancer access log event data
+
+  Attributes:
+    type (str): The type of request or connection.
+    elb (str): The resource ID of the load balancer.
+    client_ip_address (str): The IP address of the requesting client.
+    client_port (int): The port of the requesting client.
+    target_ip_address (str): The IP address of the target that processed
+        this request.
+    target_port (int): The port of the target that processed this request.
+    request_processing_time (str): The total time elapsed from
+        the time the load balancer received the request until the
+        time it sent the request to a target.
+    target_processing_time (str): The total time elapsed from
+        the time the load balancer sent the request to a target until
+        the target started to send the response headers.
+    response_processing_time (str): The total processing time elapsed.
+    target_status_code (int): The status code of the response from the target.
+    received_bytes (int): The size of the request, in bytes, received from
+        the client.
+    sent_bytes (int): The size of the response, in bytes, sent to the client.
+    user_agent (str): A User-Agent string.
+    ssl_cipher (str): [HTTPS listener] The SSL cipher.
```

Comment:
Don't use `[` => SSL cipher of the HTTPS listener

Reply:
Done